x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -151,31 +146,24 @@ string
     : STRING
     ;
 
-AND: 'AND';
-ANY: 'ANY';
-ASC: 'ASC';
-BETWEEN: 'BETWEEN';
-BY: 'BY';
-CHILD: 'CHILD';
-DESCENDANT: 'DESCENDANT';
-EVENT: 'EVENT';
-FALSE: 'FALSE';
-IN: 'IN';
-JOIN: 'JOIN';
-MAXSPAN: 'MAXSPAN';
-NOT: 'NOT';
-NULL: 'NULL';
-OF: 'OF';
-OR: 'OR';
-SEQUENCE: 'SEQUENCE';
-TRUE: 'TRUE';
-UNTIL: 'UNTIL';
-WHERE: 'WHERE';
-WITH: 'WITH';
+AND: 'and';
+BY: 'by';
```

Comment:
Isn't `ANY` a keyword?

Reply:
I've treated it as a special-case even type, but never implemented it as a keyword.
https://github.com/endgameinc/eql/blob/master/eql/schema.py#L153-L156

I've also made the `event_type "where"` part entirely optional according to the grammar, but toggleable so that you can make it required or implicitly `any where X`
https://github.com/endgameinc/eql/blob/master/eql/parser.py#L818-L821